cocos/network/CCDownloader-apple.mm

```diff
@@ -515,7 +515,18 @@ - (void)URLSession:(NSURLSession *)session downloadTask :(NSURLSessionDownloadTa
     NSString *destPath = [NSString stringWithUTF8String:storagePath];
     NSFileManager *fileManager = [NSFileManager defaultManager];
     NSURL *destURL = nil;
-    
+
+    // On iOS 9 a response with status code 4xx(Client Error) or 5xx(Server Error)
+    // might end up calling this delegate method, saving the error message to the storage path
```

Comment:
What do you mean by `might`, it's not a consistent behavior ? Maybe something else triggered the problem ?